src/bindings/interface/annotator.i

```diff
@@ -104,13 +105,10 @@
 %feature("docstring") libcellml::Annotator::assignVariableId
 "Assign an automatically generated, unique id to the given variable.";
 
-%feature("docstring") libcellml::Annotator::typeString
-"Translate the given Annotator::Type enumeration into a string.";
-
 %feature("docstring") libcellml::Annotator::duplicateCount
 "Return the number of items in the stored model with the given id.";
 
-%feature("docstring") libcellml::Annotator::isBuilt
+%feature("docstring") libcellml::Annotator::hasModel
 "Report whether the annotator index is up-to-date with its stored model state.";
 
```

Comment:
Docstring should be changed to reflect what the function does?

Reply:
Definitely.